### DISCUSSION TITLE
[Acordeón] programación con Python

### DISCUSSION THREAD

**Discussion by DanielOe21** [LaunchX-InnovaccionVirtual/CursoIntroPython#124] [Show and tell]
# Acordeón

Hola compañeros de misión!!

Hice esta libreta a manera de resumen / acordeón sobre programación con python, de ♥ espero les sirva.

https://github.com/DanielOe-21/Acordeon-de-Python

La primera parte donde dice de los ambientes virtuales es usando **Anaconda**, esta parte puede variar (y seguro así será).

No olvides marcar con ⭐ el repo! 

Follow me!
<a href="https://github.com/DanielOe-21">
<img alt="GitHub followers" src="https://img.shields.io/github/followers/DanielOe-21?style=social">
</a>

**Comment by BarajasAngel** [comment]
Gracias por el aporte ❤️

**Reply by DanielOe21** [reply]
Un placer!

**Comment by xXChAvE2Xx** [comment]
Gracias bro! Me sirvió mucho para recordar algunas cosas :)

**Reply by DanielOe21** [reply]
Con gusto ☺️
😁
